### ISSUE TITLE
netrw replacing tabs

### ISSUE THREAD

**Issue by benbotto** [vim/vim#2298]
Today I upgraded to vim 8.0.1272 and now I'm experiencing some strange bugs with tabs and netrw.  

- I open a new tab using `tabnew .`,
- navigate to a file in a new directory,
- the pen the new file.

Rather than opening the file in the newly-opened tab, the file opens in a previous tab!  Then, if I open another new tab with `tabnew .`, the tab opens in the wrong directory.  And if I open the first file, netrw navigates to a directory!

I'm sure that's quite confusing to read, so please look at the screenshot.  It demonstrates all three bugs.

![screen](https://user-images.githubusercontent.com/14243981/32461360-62f56b32-c2ea-11e7-88e4-0bf52dba83db.gif)

Thanks in advance,

[b]

**Comment by JamieMeyers** [comment]
This may be related or duplicate of https://github.com/vim/vim/issues/2141. I've added the explicit steps to reproduce there (similar to the gif above).

**Comment by benbotto** [comment]
I followed Dr. Chips instructions from #2141 as best I could.  When I run vim using his attached netrw.vimrc file I cannot reproduce the issue.  Does that mean that the issue is in fact not with netrw?  Are there any additional details I can provide?

**Comment by cecamp** [comment]
benbotto wrote:
>
> I followed Dr. Chips instructions from #2141
> <https://github.com/vim/vim/issues/2141> as best I could. When I run
> vim using his attached netrw.vimrc file I cannot reproduce the issue.
> Does that mean that the issue is in fact not with netrw? Are there any
> additional details I can provide?
>
>
In that case, something is in your .vimrc that isn't in netrw.vimrc --
ie. some setting (of course, not much is in netrw.vimrc).

 * what netrw-related options are you using?  Try figuring out which one
(if any) is involved, perhaps by adding them a few at a time to
netrw.vimrc, putting all of them in and commenting half out, etc.
 * if the problem persists with all netrw-related options you use
included, then you should continue working with detecting which
option(s) are involved.

Regards,
Chip Campbell

**Comment by JamieMeyers** [comment]
Take a look at my steps to reproduce at https://github.com/vim/vim/issues/2141#issuecomment-343534296 . I've updated them to have no .vimrc file and only setting set autochdir, which is necessary for this problem to happen.

**Comment by benbotto** [comment]
Yes, backing up @JamieMeyers comment, the issue happens to me only when autochdir is on.  Here's my vimrc.

```
set nocp
set autochdir

runtime plugin/netrwPlugin.vim
```
